mimic.py

```diff
@@ -50,11 +50,31 @@
 def mimic_dict(filename):
     """Returns mimic dict mapping each word to list of words which follow it."""
     # +++your code here+++
+    mimic_dict = {}
+    file = open(filename, 'r')
+    sentence = file.read()
+    file.close()
```

Comment:
Instead of `file = open()` and `file.close()` be sure to use the context-manager form of file opening, e.g.
`with open(filename, 'r') as f:`